observability/logging/cluster-logging-support-6x.adoc

```diff
@@ -0,0 +1,82 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="cluster-logging-support-6x"]
+include::_attributes/common-attributes.adoc[]
+= Support
+:context: cluster-logging-support-6x
+
+toc::[]
+
+include::snippets/logging-supported-config-snip.adoc[]
+include::snippets/logging-compatibility-snip.adoc[]
+include::snippets/log6x-loki-statement-snip.adoc[]
+
+{logging-uc} {for} is an opinionated collector and normalizer of application, infrastructure, and audit logs. It is intended to be used for forwarding logs to various supported systems.
+
+{logging-uc} is not:
+
+* A high scale log collection system
+* Security Information and Event Monitoring (SIEM) compliant
+* Historical or long term log retention or storage
+* A guaranteed log sink
+* Secure storage - audit logs are not stored by default
+
+[id="cluster-logging-support-CRDs_{context}"]
+== Supported API custom resource definitions
+
+The following table describes the supported Logging APIs.
+
+.Loki API support states
+[cols="3",options="header"]
+|===
+|CustomResourceDefinition (CRD)
+|ApiVersion
+|Support state
+
+|LokiStack
+|lokistack.loki.grafana.com/v1
+|Supported from 5.5
+
+|RulerConfig
+|rulerconfig.loki.grafana/v1
+|Supported from 5.7
+
+|AlertingRule
+|alertingrule.loki.grafana/v1
+|Supported from 5.7
+
+|RecordingRule
+|recordingrule.loki.grafana/v1
+|Supported from 5.7
+
+|LogFileMetricExporter
+|LogFileMetricExporter.logging.openshift.io/v1alpha1
+|Supported from 5.8
+
+|ClusterLogForwarder
+|clusterlogforwarder.logging.openshift.io/v1
+|Supported from 4.5. Removed in 6.0.
+
+|ClusterLogForwarder
+|clusterlogforwarder.observability.openshift.io/v1
+|Supported from 6.0
+|===
+
+include::modules/cluster-logging-maintenance-support-list.adoc[leveloffset=+1]
+include::modules/unmanaged-operators.adoc[leveloffset=+1]
+
+[id="support-exception-for-coo-logging-ui-plugin_{context}"]
+== Support exception for the Logging UI Plugin
+
+Until the approaching General Availability (GA) release of the Cluster Observability Operator (COO), which is currently in link:https://access.redhat.com/support/offerings/techpreview/[Technology Preview] (TP), Red{nbsp}Hat provides support to customers who are using Logging 6.0 or later with the COO for its Logging UI Plugin on {product-title} 4.14 or later. This support exception is temporary as the COO includes several independent features, some of which are still TP features, but the Logging UI Plugin is ready for GA.
```

Comment:
I believe that COO is GA as of today so we might not need this anymore. From my side this can be done in a follow up issue if you want

Reply:
Thanks, @JoaoBraveCoding, i've created https://issues.redhat.com/browse/OBSDOCS-1703 so that we can have a good history for the change.